.travis.yml

```diff
@@ -0,0 +1,40 @@
+sudo: required
```

Comment:
Cause if 'sudo: false' - ubuntu 12.04 will be used.
https://docs.travis-ci.com/user/trusty-ci-environment/